### DISCUSSION TITLE
BUG: the large model produces rounded time in timestamp

### DISCUSSION THREAD

**Discussion by Vijayabhaskar96** [openai/whisper#247] [Q&A]
The large model produces timestamps that are rounded up to the seconds, maybe integer casting somewhere in the model's output.
All other models are fine.
```
print([d["start"] for d in pred["segments"]])

[0.0, 3.0, 5.0, 8.0, 10.0, 11.0, 13.0, 15.0, 17.0, 18.0]

Checked Another audio to be sure
print([(d["start"],d["end"]) for d in pred["segments"]])

[(0.0, 4.0),
 (4.0, 7.0),
 (7.0, 12.0),
 (12.0, 17.0),
 (17.0, 19.0),
 (19.0, 22.0),
 (22.0, 26.0),
 (26.0, 29.0),
 (29.0, 34.0),
 (34.0, 37.0),
 (37.0, 40.0),
 (40.0, 43.0),
 (43.0, 46.0),
 (46.0, 49.0)]
```

**Comment by jianfch** [comment]
this has been brought up in some discussions (such as [this](https://github.com/openai/whisper/discussions/139#discussioncomment-3740861)), or in short, the predicted timestamps are biased towards integer.
